MicrosoftFluentUI.podspec

```diff
@@ -281,9 +283,21 @@ Pod::Spec.new do |s|
     core_mac.source_files = ["macos/FluentUI/Core/**/*.{swift,h}"]
   end
 
+  s.subspec 'Appearance_mac' do |appearance_mac|
+    appearance_mac.platform = :osx
+    appearance_mac.source_files = ["macos/FluentUI/Appearance/**/*.{swift,h}"]
+  end
+
+  s.subspec 'DynamicColor_mac' do |dynamiccolor_mac|
+    dynamiccolor_mac.platform = :osx
+    dynamiccolor_mac.dependency 'MicrosoftFluentUI/Appearance_mac'
+    dynamiccolor_mac.source_files = ["macos/FluentUI/DynamicColor/**/*.{swift,h}"]
+  end
+
   s.subspec 'DatePicker_mac' do |datepicker_mac|
     datepicker_mac.platform = :osx
     datepicker_mac.dependency 'MicrosoftFluentUI/Core_mac'
+    datepicker_mac.dependency 'MicrosoftFluentUI/DynamicColor_mac'
```

Comment:
Shouldn't this be **MicrosoftFluentUI/Appearance_mac**?
DatePicker does not depend on DynamicColor or ColorSet, correct?

Reply:
Did you address this?

Reply:
Apologies, the push failed. This should be updated now.